_layouts/welcome.html

```diff
@@ -0,0 +1,3 @@
+<h1>{{ title }}</h1>
```

Comment:
We will need this code block on top of the file
This is telling Jekyll this even exists as a layout.
```
---
layout: default
---
```